source/stylesheets/base/_feature_sections.scss

```diff
@@ -21,32 +21,44 @@
 
     .illustration {
       align-items: flex-end;
-      justify-content: center;
+      justify-content: flex-end;
     }
 
     .text-and-description {
       @include grid-column(6 of 12);
       align-items: center;
       display: flex;
     }
+
+    h1 {
+      font-size: 3.1em;
+    }
+
+    time {
```

Comment:
The same thing applies here. What about something like, `.jumbo-feature-time`?

Reply:
This will also all you to unnest things. That will result in cleaner output css.